src/js/api/api.js

```diff
@@ -967,6 +967,14 @@ Object.assign(Api.prototype, /** @lends Api.prototype */ {
     },
 
     /**
+     * Extracts the video tag in the foreground.
+     * @returns {string} videoTag - the video tag in the foreground.
+     */
+    getVideoTag() {
+        return this.getContainer().getElementsByClassName('jw-media')[0].getElementsByTagName('video')[0];
```

Comment:
What happens if there's no container, or `jw-media` containers, or a `video` within the container?

Reply:
The following works: 
```
const container = this.getContainer();
return container && container.querySelector('video'); // returns the 1st video element or null if there's none
```

Reply:
addressed